### DISCUSSION TITLE
Event Suggestions

### DISCUSSION THREAD

**Discussion by CarsonKompon** [CarsonKompon/sbox-plates#1] [Ideas]
If you have an idea for a new Event for Pl&tes, leave a reply below this post.

If you see a reply with an idea you really like, give it a thumbs up 👍

**Comment by Bugtogs** [comment]
A random or small chance to be cursed with the troll face for 10 or 20 seconds. Makes everything black and white and covers 70 percent of the screen in different variants of troll face whilst playing the troll theme.

**Reply by Bugtogs** [reply]
A sniper rifle that insta-kills, but only has one bullet, and you can only kill a specific target, if you cant kill the target you get a random bad event on your plate, or you get sniped
